coalib/misc/CachingUtilities.py

```diff
@@ -64,14 +65,13 @@ def pickle_load(log_printer, identifier, fallback=None):
 
     >>> from pyprint.NullPrinter import NullPrinter
     >>> from coalib.output.printers.LogPrinter import LogPrinter
-    >>> log_printer = LogPrinter(NullPrinter())
     >>> test_data = {"answer": 42}
-    >>> pickle_dump(log_printer, "test_project", test_data)
+    >>> pickle_dump(None, "test_project", test_data)
     True
-    >>> pickle_load(log_printer, "test_project")
+    >>> pickle_load(None, "test_project")
     {'answer': 42}
-    >>> pickle_load(log_printer, "nonexistent_project")
-    >>> pickle_load(log_printer, "nonexistent_project", fallback=42)
+    >>> pickle_load(None, "nonexistent_project")
+    >>> pickle_load(None, "nonexistent_project", fallback=42)
```

Comment:
we need to file an issue afterwards changing the double-quotes to single quotes :+1:

Reply:
Filed at https://github.com/coala/coala/issues/4674